reconcile/queries.py

```diff
@@ -1944,11 +1945,36 @@ def get_pipelines_providers():
 
 
 def get_jira_boards():
-    """ Returns Jira boards resources defined in app-interface """
+    """Returns Jira boards resources defined in app-interface """
     gqlapi = gql.get_api()
     return gqlapi.query(JIRA_BOARDS_QUERY)['jira_boards']
 
 
+JIRA_BOARDS_QUICK_QUERY = """
+{
+  jira_boards: jira_boards_v1 (path: "APATH") {
+    path
+    name
+    server {
+      serverUrl
+      token {
+        path
+        field
+        version
+        format
+      }
+    }
+  }
+}
+"""
+
+
+def get_simple_jira_boards(app_path: str):
+    gqlapi = gql.get_api()
+    query = JIRA_BOARDS_QUICK_QUERY.replace("APATH", shlex.quote(app_path))
```

Comment:
The restriction is that the query has a lot of curly brackets already. I could use `%s` instead, but since it's been deprecated for a long time I settled with replace.